configure

```diff
@@ -44,8 +44,8 @@ while getopts ":ho:-:p" opt; do
 	    # process --long-options
 	    case "$OPTARG" in
                 help)  usage ;;
-                *=*)   opt=${OPTARG%%=*}; val=${OPTARG#*=}
-                       eval "$opt=$val" ;;
+                *=*)   opt=${OPTARG%=*}; val=${OPTARG#*=}
```

Comment:
Any particular reason for changing `%%` to `%`? The `%%` seems preferable, since it should be possible for option values to contain `=` themselves, i.e. `--prefix=/my/weird=dir/`. Maybe not likely to be needed unless more general flags are added down the road, but afaict it doesn't hurt to keep it.